app/Organizer/OrganizerControllerProvider.php

```diff
@@ -154,12 +154,21 @@ public function register(Application $app): void
             fn (Application $application) => new DeleteLabelRequestHandler($app['event_command_bus'])
         );
 
-        $app['organizer_permissions_controller'] = $app->share(
+        $app[GetPermissionsForCurrentUserRequestHandler::class] = $app->share(
             function (Application $app) {
-                return new OfferPermissionsController(
+                return new GetPermissionsForCurrentUserRequestHandler(
                     [Permission::organisatiesBewerken()],
                     $app['organizer_permission_voter'],
-                    $app['current_user_id'] ? new StringLiteral($app['current_user_id']) : null
+                    $app['current_user_id'] ?? null
```

Comment:
Note: This service does not exist anymore since I merged #1106 . You can get the current user id by getting the `CurrentUser::class` service and then calling the `getId()` method. You can either inject the `CurrentUser` object or a `string` id in the request handler.